stable/aws-for-fluent-bit/README.md

```diff
@@ -38,7 +38,10 @@ helm delete aws-for-fluent-bit --namespace kube-system
 | `imagePullSecrets` | Docker registry pull secret | `[]` |
 | `serviceAccount.create` | Whether a new service account should be created | `true` |
 | `serviceAccount.name` | Name of the service account | `aws-for-fluent-bit` |
-| `service.extraService` | Append to existing service with this value | `""` |
+| `k8sService.type`| Type of service to be created - options are ClusterIP, NodePort, LoadBalancer |`ClusterIP`|
+| `k8sService.port`| TCP port of the serviceMonitor service. | 2020 |
+| `k8sService.targetPort`| TCP targetPort for service to connect to fluent-bit. | 2020 |
```

Comment:
Sorry I am still very confused by this change. Apologies for how much time its taking for you to help me to understand, but I need to understand before I can merge it.

1. these options should go down next to the other `serviceMonitor` options right? Since logically they are related.
2. There are two new namespaces here for options for monitoring FLB with the service monitor- `k8sService.*` and `serviceMonitor.*`- why is that? Could it all just be one namespace, `serviceMonitor.*`? Is there some logical distinction between these? From looking at the changes, it look like the `k8sService` ones are ports exposed by FLB for use by the service monitoring? I will make a comment on those yaml changes. If my understanding is correct, should we use the same `serviceMonitor` options for these?

Reply:
@razorsk8jz Sorry, I am confused, so the k8sService is separate from serviceMonitor? The description here says serviceMonitor. Sorry, please explain exactly what these do to me, and I think the help text should be longer and more descriptive for these options. 

>  the k8sService is not logically tied to the serviceMonitor as it is also needed for health check 

I don't understand how its not tied to it. When I check how these configs affect the yaml, it looks like they only affect service monitor, not Fluent bit. I will make a comment on the yaml.